scripts/trigger-solution-tests.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+function watch_build() {
+  local -r solution="$1"
+  local -r build_id="$2"
+
+  local build_status=""
+  local -a args=(
+    --filter
+    "ID:${build_id}"
+    --format
+    "value(status)"
+  )
+
+  if [[ "${solution}" == k8s* ]]; then
+    args+=(
+      --region
+      us-central1
+    )
+  fi
+
+  while true; do
+    build_status="$(gcloud builds list "${args[@]}")"
```

Comment:
Can't we use `gcloud builds describe $build_id` ?

That might be clearer than list and filter